client/src/components/TestDetails.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+function renderTest(test) {
+  return (
+    <React.Fragment key={parseInt(test.id._text)}>
```

Comment:
`test.id._text` should be stored in store like this. It should be converted into proper format in the saga/reducer

Reply:
I mean the whole data from testman, not only this field. Keeping test_id as string should be fine